lib/hexo/default_config.js

```diff
@@ -65,9 +65,8 @@ module.exports = {
   deploy: {},
 
   // ignore files from processing
-  ignore: [],
+  ignore: ['**/themes/*/node_modules/**', '**/themes/*/.git/**'],
```

Comment:
After default `ignore` array declared in `lib/box/index.js`, we could use empty array then.

```suggestion
  ignore: [],
```